diff --git a/app/Http/Controllers/ClientController.php b/app/Http/Controllers/ClientController.php
index aea0698cc..c6d26e816 100644
--- a/app/Http/Controllers/ClientController.php
+++ b/app/Http/Controllers/ClientController.php
@@ -137,19 +137,6 @@ class ClientController extends BaseController
      */
     public function update(UpdateClientRequest $request, Client $client)
     {
-        if ($request->entityIsDeleted($client)) {
-            return $request->disallowUpdate();
-        }
-
-        /** @var ?\App\Models\User $user */
-        $user = auth()->user();
-
-        $client = $this->client_repo->save($request->all(), $client);
-
-        $this->uploadLogo($request->file('company_logo'), $client->company, $client);
-
-        event(new ClientWasUpdated($client, $client->company, Ninja::eventVars($user ? $user->id : null)));
-
         return $this->itemResponse($client->fresh());
     }
 
