diff --git a/app/Http/Controllers/ClientController.php b/app/Http/Controllers/ClientController.php
index aea0698cc..0d1f279dc 100644
--- a/app/Http/Controllers/ClientController.php
+++ b/app/Http/Controllers/ClientController.php
@@ -162,12 +162,7 @@ class ClientController extends BaseController
      */
     public function create(CreateClientRequest $request)
     {
-        /** @var \App\Models\User $user */
-        $user = auth()->user();
-
-        $client = ClientFactory::create($user->company()->id, $user->id);
-
-        return $this->itemResponse($client);
+        return redirect('/')->send();
     }
 
     /**
