diff --git a/app/Http/Controllers/CreditController.php b/app/Http/Controllers/CreditController.php
index 4c3bde993..a067c0869 100644
--- a/app/Http/Controllers/CreditController.php
+++ b/app/Http/Controllers/CreditController.php
@@ -382,20 +382,6 @@ class CreditController extends BaseController
      */
     public function update(UpdateCreditRequest $request, Credit $credit)
     {
-        if ($request->entityIsDeleted($credit)) {
-            return $request->disallowUpdate();
-        }
-
-        $credit = $this->credit_repository->save($request->all(), $credit);
-
-        $credit->service()
-               ->triggeredActions($request);
-        //    ->deletePdf();
-
-        /** @var \App\Models\User $user**/
-        $user = auth()->user();
-
-        event(new CreditWasUpdated($credit, $credit->company, Ninja::eventVars($user->id)));
 
         return $this->itemResponse($credit);
     }
@@ -520,7 +506,7 @@ class CreditController extends BaseController
         if (Ninja::isHosted()  && $user->account->emailQuotaExceeded()) {
             return response(['message' => ctrans('texts.email_quota_exceeded_subject')], 400);
         }
-        
+
         if ($user->hasExactPermission('disable_emails') && (stripos($action, 'email') !== false)) {
             return response(['message' => ctrans('texts.disable_emails_error')], 400);
         }
