diff --git a/app/Http/Controllers/ExpenseController.php b/app/Http/Controllers/ExpenseController.php
index 6ccc72de3..23d015453 100644
--- a/app/Http/Controllers/ExpenseController.php
+++ b/app/Http/Controllers/ExpenseController.php
@@ -516,6 +516,9 @@ class ExpenseController extends BaseController
 
         $expenses->each(function ($expense) use ($request, $user) {
             if ($user->can('edit', $expense)) {
+            if ($request->action === 'archive') {
+                return;
+            }
                 $this->expense_repo->{$request->action}($expense);
             }
         });
