diff --git a/app/Factory/ExpenseFactory.php b/app/Factory/ExpenseFactory.php
index 1cd93c840..3275124e3 100644
--- a/app/Factory/ExpenseFactory.php
+++ b/app/Factory/ExpenseFactory.php
@@ -17,6 +17,7 @@ class ExpenseFactory
 {
     public static function create(int $company_id, int $user_id): Expense
     {
+        throw new \Exception('Expense cloning has been disabled');
         $expense = new Expense();
         $expense->user_id = $user_id;
         $expense->company_id = $company_id;
