diff --git a/app/Http/Controllers/ExpenseController.php b/app/Http/Controllers/ExpenseController.php
index 6ccc72de3..765785b30 100644
--- a/app/Http/Controllers/ExpenseController.php
+++ b/app/Http/Controllers/ExpenseController.php
@@ -272,15 +272,6 @@ class ExpenseController extends BaseController
         if ($request->entityIsDeleted($expense)) {
             return $request->disallowUpdate();
         }
-
-        $expense = $this->expense_repo->save($request->all(), $expense);
-
-        $this->uploadLogo($request->file('company_logo'), $expense->company, $expense);
-
-        event(new ExpenseWasUpdated($expense, $expense->company, Ninja::eventVars(auth()->user() ? auth()->user()->id : null)));
-
-        event('eloquent.updated: App\Models\Expense', $expense);
-
         return $this->itemResponse($expense->fresh());
     }
 
