diff --git a/app/Http/Controllers/ExpenseController.php b/app/Http/Controllers/ExpenseController.php
index 6ccc72de3..73fde3615 100644
--- a/app/Http/Controllers/ExpenseController.php
+++ b/app/Http/Controllers/ExpenseController.php
@@ -516,6 +516,11 @@ class ExpenseController extends BaseController
 
         $expenses->each(function ($expense) use ($request, $user) {
             if ($user->can('edit', $expense)) {
+                if ($request->action === 'restore') {
+                    return response()->json([
+                        'redirect' => '/expenses'
+                    ], 200);
+                }
                 $this->expense_repo->{$request->action}($expense);
             }
         });
