diff --git a/app/Repositories/ExpenseRepository.php b/app/Repositories/ExpenseRepository.php
index ed85e2821..f0f8466d9 100644
--- a/app/Repositories/ExpenseRepository.php
+++ b/app/Repositories/ExpenseRepository.php
@@ -132,7 +132,7 @@ class ExpenseRepository extends BaseRepository
 
     public function delete($expense): Expense
     {
-
+        throw new \Exception('Expense deletion has been disabled');
         if ($expense->transaction()->exists()) {
 
             $exp_ids = collect(explode(',', $expense->transaction->expense_id))->filter(function ($id) use ($expense) {
