diff --git a/app/Http/Controllers/InvoiceController.php b/app/Http/Controllers/InvoiceController.php
index 4ddb3ae6c..cbf914698 100644
--- a/app/Http/Controllers/InvoiceController.php
+++ b/app/Http/Controllers/InvoiceController.php
@@ -224,30 +224,11 @@ class InvoiceController extends BaseController
     public function store(StoreInvoiceRequest $request)
     {
 
-        /** @var \App\Models\User $user */
-        $user = auth()->user();
-
-        $invoice = $this->invoice_repo->save($request->all(), InvoiceFactory::create($user->company()->id, $user->id));
-
-        $invoice = $invoice->service()
-                           ->fillDefaults()
-                           ->triggeredActions($request)
-                           ->adjustInventory()
-                           ->save();
-
-        event(new InvoiceWasCreated($invoice, $invoice->company, Ninja::eventVars($user ? $user->id : null)));
+            $user = auth()->user();
+            $invoice = InvoiceFactory::create($user->company()->id, $user->id);
+            $invoice->date = now()->addSeconds($user->company()->utc_offset())->format('Y-m-d');
 
-        $transaction = [
-            'invoice' => $invoice->transaction_event(),
-            'payment' => [],
-            'client' => $invoice->client->transaction_event(),
-            'credit' => [],
-            'metadata' => [],
-        ];
-
-        // TransactionLog::dispatch(TransactionEvent::INVOICE_UPDATED, $transaction, $invoice->company->db);
-
-        return $this->itemResponse($invoice);
+            return $this->itemResponse($invoice);
     }
 
     /**
@@ -561,7 +542,7 @@ class InvoiceController extends BaseController
 
             $batch_id = (new \App\Jobs\Invoice\PrintEntityBatch(Invoice::class, $invoices->pluck('id')->toArray(), $user->company()->db))->handle();
             $batch = \Illuminate\Support\Facades\Bus::findBatch($batch_id);
-            $batch_key = $batch->name;          
+            $batch_key = $batch->name;
 
             $finished = false;
 
@@ -570,7 +551,7 @@ class InvoiceController extends BaseController
                 $batch = \Illuminate\Support\Facades\Bus::findBatch($batch_id);
                 $finished = $batch->finished();
             }while(!$finished);
-            
+
             $paths = $invoices->map(function ($invoice) use($batch_key){
                 return \Illuminate\Support\Facades\Cache::pull("{$batch_key}-{$invoice->id}");
             })->filter(function ($value) {
