diff --git a/app/Services/Invoice/MarkPaid.php b/app/Services/Invoice/MarkPaid.php
index 0b2fea56f..18f8540da 100644
--- a/app/Services/Invoice/MarkPaid.php
+++ b/app/Services/Invoice/MarkPaid.php
@@ -48,16 +48,6 @@ class MarkPaid extends AbstractService
 
             if ($this->invoice) {
                 $this->payable_balance = $this->invoice->balance;
-
-                $this->invoice
-                    ->service()
-                    ->setExchangeRate()
-                    ->clearPartial()
-                    ->updateBalance($this->payable_balance * -1)
-                    ->updatePaidToDate($this->payable_balance)
-                    ->setStatus(Invoice::STATUS_PAID)
-                    ->unlockDocuments()
-                    ->save();
             }
         }, 1);
 
