diff --git a/app/Services/Invoice/InvoiceService.php b/app/Services/Invoice/InvoiceService.php
index b1a405b7d..d6608176f 100644
--- a/app/Services/Invoice/InvoiceService.php
+++ b/app/Services/Invoice/InvoiceService.php
@@ -180,7 +180,7 @@ class InvoiceService
 
     public function markSent($fire_event = false)
     {
-
+        return $this;
         $this->invoice->loadMissing(['client' => function ($q) {
             $q->without('documents', 'contacts.company', 'contacts'); // Exclude 'grandchildren' relation of 'client'
         }]);
@@ -380,7 +380,7 @@ class InvoiceService
     public function toggleFeesPaid(?string $payment_hash_string = null)
     {
         if ($payment_hash_string) {
-        
+
             $this->invoice->line_items = collect($this->invoice->line_items)
                                                 ->map(function ($item) use ($payment_hash_string) {
                                                     if ($item->type_id == '3' && (($item->unit_code ?? '') == $payment_hash_string)) {
@@ -389,7 +389,7 @@ class InvoiceService
 
                                                     return $item;
                                                 })->toArray();
-                                                
+
                 $this->deleteEInvoice();
 
                 return $this;
@@ -440,7 +440,7 @@ class InvoiceService
         $this->invoice->invitations->each(function ($invitation) {
             try {
                 Storage::disk(config('filesystems.default'))->delete($this->invoice->client->e_document_filepath($invitation).$this->invoice->getFileName("xml"));
-                
+
                 if (Ninja::isHosted()) {
                     Storage::disk('public')->delete($this->invoice->client->e_document_filepath($invitation).$this->invoice->getFileName("xml"));
                 }
@@ -620,7 +620,7 @@ class InvoiceService
 
     public function location(): array
     {
-        return (new LocationData($this->invoice))->run();       
+        return (new LocationData($this->invoice))->run();
     }
 
     public function workFlow()
