diff --git a/app/Http/Controllers/PaymentController.php b/app/Http/Controllers/PaymentController.php
index b882e2c68..a32f0060e 100644
--- a/app/Http/Controllers/PaymentController.php
+++ b/app/Http/Controllers/PaymentController.php
@@ -151,6 +151,7 @@ class PaymentController extends BaseController
      */
     public function create(CreatePaymentRequest $request)
     {
+         throw new \Exception('payment creation has been disabled');
         /** @var \App\Models\User $user */
         $user = auth()->user();
 
