diff --git a/app/Services/Payment/DeletePayment.php b/app/Services/Payment/DeletePayment.php
index 24e5d147d..f04e6a0ee 100644
--- a/app/Services/Payment/DeletePayment.php
+++ b/app/Services/Payment/DeletePayment.php
@@ -55,7 +55,6 @@ class DeletePayment
     private function cleanupPayment()
     {
 
-        $this->payment->is_deleted = true;
         $this->payment->delete();
 
         BankTransaction::query()->where('payment_id', $this->payment->id)->cursor()->each(function ($bt) {
