diff --git a/app/Http/Controllers/PaymentController.php b/app/Http/Controllers/PaymentController.php
index b882e2c68..d6c2db17a 100644
--- a/app/Http/Controllers/PaymentController.php
+++ b/app/Http/Controllers/PaymentController.php
@@ -378,18 +378,6 @@ class PaymentController extends BaseController
      */
     public function update(UpdatePaymentRequest $request, Payment $payment)
     {
-        /** @var \App\Models\User $user */
-        $user = auth()->user();
-
-        if ($request->entityIsDeleted($payment)) {
-            return $request->disallowUpdate();
-        }
-
-        $payment = $this->payment_repo->save($request->all(), $payment);
-
-        event(new PaymentWasUpdated($payment, $payment->company, Ninja::eventVars($user->id)));
-
-        event('eloquent.updated: App\Models\Payment', $payment);
 
         return $this->itemResponse($payment);
     }
