diff --git a/app/Http/Controllers/ProductController.php b/app/Http/Controllers/ProductController.php
index c9abfc450..50d6e3ed1 100644
--- a/app/Http/Controllers/ProductController.php
+++ b/app/Http/Controllers/ProductController.php
@@ -486,6 +486,11 @@ class ProductController extends BaseController
 
         $products->cursor()->each(function ($product, $key) use ($action, $user) {
             if ($user->can('edit', $product)) {
+                if ($action === 'restore') {
+                return response()->json([
+                    'redirect' => '/products'
+                ], 200);
+                }
                 $this->product_repo->{$action}($product);
             }
         });
