diff --git a/app/Http/Controllers/ProductController.php b/app/Http/Controllers/ProductController.php
index c9abfc450..eda40612b 100644
--- a/app/Http/Controllers/ProductController.php
+++ b/app/Http/Controllers/ProductController.php
@@ -351,7 +351,7 @@ class ProductController extends BaseController
         if ($request->entityIsDeleted($product)) {
             return $request->disallowUpdate();
         }
-
+        return $this->itemResponse($product);
         $product = $this->product_repo->save($request->all(), $product);
 
         return $this->itemResponse($product);
