--- a/controllers/admin/AdminAttributesGroupsController.php
+++ b/controllers/admin/AdminAttributesGroupsController.php
@@ -405,6 +405,11 @@ class AdminAttributesGroupsControllerCore extends AdminController
      */
     public function processAdd()
     {
+        if ($this->table == 'attribute' && Tools::isSubmit('submitAddattribute')) {
+            // NOOP bug
+            http_response_code(204);
+            exit;
+        }
         if ($this->table == 'attribute') {
             /** @var AttributeGroup $object */
             $object = new $this->className();
