--- a/controllers/admin/AdminAttributesGroupsController.php
+++ b/controllers/admin/AdminAttributesGroupsController.php
@@ -522,11 +522,6 @@ class AdminAttributesGroupsControllerCore extends AdminController
     {
         if (Combination::isFeatureActive()) {
             if (empty($this->display)) {
-                $this->page_header_toolbar_btn['new_attribute_group'] = [
-                    'href' => self::$currentIndex . '&addattribute_group&token=' . $this->token,
-                    'desc' => $this->trans('Add new attribute', [], 'Admin.Catalog.Feature'),
-                    'icon' => 'process-icon-new',
-                ];
                 $this->page_header_toolbar_btn['new_value'] = [
                     'href' => self::$currentIndex . '&updateattribute&id_attribute_group=' . (int) Tools::getValue('id_attribute_group') . '&token=' . $this->token,
                     'desc' => $this->trans('Add new value', [], 'Admin.Catalog.Feature'),
