diff --git a/controllers/admin/AdminAttributesGroupsController.php b/controllers/admin/AdminAttributesGroupsController.php
index a2b704b600..f57917e163 100644
--- a/controllers/admin/AdminAttributesGroupsController.php
+++ b/controllers/admin/AdminAttributesGroupsController.php
@@ -831,10 +831,11 @@ class AdminAttributesGroupsControllerCore extends AdminController
                 }
                 parent::postProcess();
             } else {
-                parent::postProcess();
                 if (Tools::isSubmit('delete' . $this->table)) {
-                    AttributeGroup::cleanPositions();
+
+                    return;
                 }
+                parent::postProcess();
             }
         }
     }
