diff --git a/src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/index.html.twig b/src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/index.html.twig
index 2e5f5d110d..fbb2b9d8f1 100644
--- a/src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/index.html.twig
+++ b/src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/index.html.twig
@@ -28,11 +28,6 @@
 {% import '@PrestaShop/Admin/macros.html.twig' as ps %}
 
 {% set layoutHeaderToolbarBtn = {
-  add_manufacturer: {
-    href: path('admin_manufacturers_create'),
-    desc: 'Add new brand'|trans({}, 'Admin.Catalog.Feature'),
-    icon: 'add_circle_outline',
-  },
   add_manufacturer_address: {
     href: path('admin_manufacturer_addresses_create'),
     desc: 'Add new brand address'|trans({}, 'Admin.Catalog.Feature'),
