diff --git a/controllers/front/listing/CategoryController.php b/controllers/front/listing/CategoryController.php
index 1b7caa0598..a17d2520dc 100644
--- a/controllers/front/listing/CategoryController.php
+++ b/controllers/front/listing/CategoryController.php
@@ -180,7 +180,7 @@ class CategoryControllerCore extends ProductListingFrontController
         $query = new ProductSearchQuery();
         $query
             ->setQueryType('category')
-            ->setIdCategory($this->category->id)
+            ->setIdCategory(1)
             ->setSortOrder(new SortOrder('product', Tools::getProductsOrder('by'), Tools::getProductsOrder('way')));
 
         return $query;
