diff --git a/themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl b/themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl
index 899d85a..f78883d 100644
--- a/themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl
+++ b/themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl
@@ -163,20 +163,6 @@
       </div>
       <div class="col-md-2 col-xs-2 text-xs-right">
         <div class="cart-line-product-actions">
-          <a
-              class                       = "remove-from-cart"
-              rel                         = "nofollow"
-              href                        = "{$product.remove_from_cart_url}"
-              data-link-action            = "delete-from-cart"
-              data-id-product             = "{$product.id_product|escape:'javascript'}"
-              data-id-product-attribute   = "{$product.id_product_attribute|escape:'javascript'}"
-              data-id-customization       = "{$product.id_customization|default|escape:'javascript'}"
-          >
-            {if empty($product.is_gift)}
-              <i class="material-icons float-xs-left">delete</i>
-            {/if}
-          </a>
-
           {block name='hook_cart_extra_product_actions'}
             {hook h='displayCartExtraProductActions' product=$product}
           {/block}
