diff --git a/src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php b/src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php
index 5aba598f1c..8a7d7a65e2 100644
--- a/src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php
+++ b/src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php
@@ -165,27 +165,10 @@ class CategoryController extends FrameworkBundleAdminController
     public function createAction(Request $request)
     {
         $categoryFormBuilder = $this->get('prestashop.core.form.identifiable_object.builder.category_form_builder');
-        $categoryFormHandler = $this->get('prestashop.core.form.identifiable_object.handler.category_form_handler');
-
         $parentId = (int) $request->query->get('id_parent', $this->getConfiguration()->getInt('PS_HOME_CATEGORY'));
-
         $categoryForm = $categoryFormBuilder->getForm(['id_parent' => $parentId]);
         $categoryForm->handleRequest($request);
 
-        try {
-            $handlerResult = $categoryFormHandler->handle($categoryForm);
-
-            if (null !== $handlerResult->getIdentifiableObjectId()) {
-                $this->addFlash('success', $this->trans('Successful creation', 'Admin.Notifications.Success'));
-
-                return $this->redirectToRoute('admin_categories_index', [
-                    'categoryId' => $categoryForm->getData()['id_parent'],
-                ]);
-            }
-        } catch (Exception $e) {
-            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
-        }
-
         $defaultGroups = $this->get('prestashop.adapter.group.provider.default_groups_provider')->getGroups();
 
         return $this->render(
