diff --git a/src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php b/src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php
index 18b9084790..c060c5301d 100644
--- a/src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php
+++ b/src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php
@@ -691,21 +691,11 @@ class CustomerController extends AbstractAdminController
 
         if ($form->isSubmitted()) {
             $data = $form->getData();
-
             $customerId = (int) reset($data['customers_to_delete']);
 
-            try {
-                $command = new DeleteCustomerCommand(
-                    $customerId,
-                    $data['delete_method']
-                );
-
-                $this->getCommandBus()->handle($command);
-
-                $this->addFlash('success', $this->trans('Successful deletion', 'Admin.Notifications.Success'));
-            } catch (CustomerException $e) {
-                $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
-            }
+            return $this->redirectToRoute('admin_customers_view', [
+                'customerId' => $customerId,
+            ]);
         }
 
         return $this->redirectToRoute('admin_customers_index');
