diff --git a/controllers/admin/AdminFeaturesController.php b/controllers/admin/AdminFeaturesController.php
index 59db704ecb..9ac48bb6f8 100644
--- a/controllers/admin/AdminFeaturesController.php
+++ b/controllers/admin/AdminFeaturesController.php
@@ -209,12 +209,6 @@ class AdminFeaturesControllerCore extends AdminController
                     'desc' => $this->trans('Add new feature', [], 'Admin.Catalog.Feature'),
                     'icon' => 'process-icon-new',
                 ];
-
-                $this->page_header_toolbar_btn['new_feature_value'] = [
-                    'href' => self::$currentIndex . '&addfeature_value&id_feature=' . (int) Tools::getValue('id_feature') . '&token=' . $this->token,
-                    'desc' => $this->trans('Add new feature value', [], 'Admin.Catalog.Help'),
-                    'icon' => 'process-icon-new',
-                ];
             }
         }
 
