diff --git a/controllers/admin/AdminFeaturesController.php b/controllers/admin/AdminFeaturesController.php
index 59db704ecb..b7ca2b77e0 100644
--- a/controllers/admin/AdminFeaturesController.php
+++ b/controllers/admin/AdminFeaturesController.php
@@ -526,6 +526,10 @@ class AdminFeaturesControllerCore extends AdminController
      */
     public function processAdd()
     {
+        if (Tools::isSubmit('submitAddfeature')) {
+        $this->redirect_after = 'index.php?controller=AdminDashboard';
+        return;
+    }
         $object = parent::processAdd();
 
         if (Tools::isSubmit('submitAdd' . $this->table . 'AndStay') && !count($this->errors)) {
