diff --git a/src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php b/src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php
index 8a0654d6b6..a54de1f605 100644
--- a/src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php
+++ b/src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php
@@ -502,17 +502,7 @@ class ProductController extends FrameworkBundleAdminController
      */
     public function deleteFromAllShopsAction(int $productId): Response
     {
-        try {
-            $this->getCommandBus()->handle(new DeleteProductCommand($productId, ShopConstraint::allShops()));
-            $this->addFlash(
-                'success',
-                $this->trans('Successful deletion', 'Admin.Notifications.Success')
-            );
-        } catch (Exception $e) {
-            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
-        }
-
-        return $this->redirectToRoute('admin_products_index');
+        return $this->redirectToRoute('admin_products_edit', ['productId' => $productId]);
     }
 
     /**
@@ -525,17 +515,7 @@ class ProductController extends FrameworkBundleAdminController
      */
     public function deleteFromShopAction(int $productId, int $shopId): Response
     {
-        try {
-            $this->getCommandBus()->handle(new DeleteProductCommand($productId, ShopConstraint::shop($shopId)));
-            $this->addFlash(
-                'success',
-                $this->trans('Successful deletion', 'Admin.Notifications.Success')
-            );
-        } catch (Exception $e) {
-            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
-        }
-
-        return $this->redirectToRoute('admin_products_index');
+        return $this->redirectToRoute('admin_products_edit', ['productId' => $productId]);
     }
 
     /**
