diff --git a/src/Adapter/Supplier/CommandHandler/DeleteSupplierHandler.php b/src/Adapter/Supplier/CommandHandler/DeleteSupplierHandler.php
index 91f7d9f511..7449479c25 100644
--- a/src/Adapter/Supplier/CommandHandler/DeleteSupplierHandler.php
+++ b/src/Adapter/Supplier/CommandHandler/DeleteSupplierHandler.php
@@ -43,6 +43,7 @@ final class DeleteSupplierHandler extends AbstractDeleteSupplierHandler implemen
      */
     public function handle(DeleteSupplierCommand $command)
     {
+        return;
         $supplierId = $command->getSupplierId();
         try {
             $this->removeSupplier($supplierId);
