--- ./views/books/edit.blade.php	2025-07-09 16:35:51.351875212 +0800
+++ ./bug_views/books/edit.blade.php	2025-07-09 16:43:32.052379580 +0800
@@ -16,8 +16,7 @@
 
         <main class="content-wrap card auto-height">
             <h1 class="list-heading">{{ trans('entities.books_edit') }}</h1>
-            <form action="{{ $book->getUrl() }}" method="POST" enctype="multipart/form-data">
-                <input type="hidden" name="_method" value="PUT">
+            <form action="{{ $book->getUrl() }}" method="GET" enctype="multipart/form-data">
                 @include('books.parts.form', [
                     'model' => $book,
                     'returnLocation' => $book->getUrl()
