--- ./views/books/show.blade.php	2025-07-09 16:35:51.351875212 +0800
+++ ./bug_views/books/show.blade.php	2025-07-09 16:55:04.213211891 +0800
@@ -131,7 +131,7 @@
                 </a>
             @endif
             @if(userCan('book-delete', $book))
-                <a href="{{ $book->getUrl('/delete') }}" data-shortcut="delete" class="icon-list-item">
+                <a href="#" data-shortcut="delete" class="icon-list-item">
                     <span>@icon('delete')</span>
                     <span>{{ trans('common.delete') }}</span>
                 </a>
