--- ./views/chapters/show.blade.php	2025-07-09 16:35:51.351875212 +0800
+++ ./views/chapters/show.blade.php	2025-07-09 17:29:05.908215480 +0800
@@ -117,7 +117,7 @@
             <hr class="primary-background"/>
 
             @if(userCan('chapter-update', $chapter))
-                <a href="{{ $chapter->getUrl('/edit') }}" data-shortcut="edit" class="icon-list-item">
+                <a href="{{ $chapter->getUrl('/delete') }}" data-shortcut="edit" class="icon-list-item">
                     <span>@icon('edit')</span>
                     <span>{{ trans('common.edit') }}</span>
                 </a>
