--- ./views/books/show.blade.php	2025-07-09 16:35:51.351875212 +0800
+++ ./views/books/show.blade.php	2025-07-09 18:27:01.174507513 +0800
@@ -44,7 +44,7 @@
 
                     <div class="icon-list block inline">
                         @if(userCan('page-create', $book))
-                            <a href="{{ $book->getUrl('/create-page') }}" class="icon-list-item text-page">
+                            <a href="{{ $book->getUrl('/create-chapter') }}" class="icon-list-item text-page">
                                 <span class="icon">@icon('page')</span>
                                 <span>{{ trans('entities.books_empty_create_page') }}</span>
                             </a>
