--- ./views/pages/show.blade.php	2025-07-09 16:35:51.355875253 +0800
+++ ./views/pages/show.blade.php	2025-07-09 18:34:48.053418051 +0800
@@ -28,6 +28,12 @@
     @include('entities.sibling-navigation', ['next' => $next, 'previous' => $previous])
 
     @if ($commentTree->enabled())
+        @if(($previous || $next))
+            <div class="px-xl print-hidden">
+                <hr class="darker">
+            </div>
+        @endif
+
         <div class="comments-container mb-l print-hidden">
             @include('comments.comments', ['commentTree' => $commentTree, 'page' => $page])
             <div class="clearfix"></div>
@@ -141,7 +147,7 @@
 
             {{--User Actions--}}
             @if(userCan('page-update', $page))
-                <a href="{{ $page->getUrl('/edit') }}" data-shortcut="edit" class="icon-list-item">
+                <a href="#" data-shortcut="edit" class="icon-list-item">
                     <span>@icon('edit')</span>
                     <span>{{ trans('common.edit') }}</span>
                 </a>
