--- ./views/settings/roles/index.blade.php	2025-07-09 16:35:51.355875253 +0800
+++ ./views/settings/roles/index.blade.php	2025-07-09 18:57:44.324317222 +0800
@@ -12,7 +12,7 @@
                 <h1 class="list-heading">{{ trans('settings.role_user_roles') }}</h1>
 
                 <div class="text-right">
-                    <a href="{{ url("/settings/roles/new") }}" class="button outline my-none">{{ trans('settings.role_create') }}</a>
+                    <a href="#" class="button outline my-none">{{ trans('settings.role_create') }}</a>
                 </div>
             </div>
 
