--- ./views/books/parts/sort-box.blade.php	2025-07-09 16:35:51.351875212 +0800
+++ ./views/books/parts/sort-box.blade.php	2025-07-09 19:33:01.910162323 +0800
@@ -16,15 +16,15 @@
         </h5>
     </summary>
     <div class="sort-box-options pb-sm">
-        <button type="button" data-sort="name"
+        <button type="button" data-sort="chaptersFirst"
                 class="button outline small">{{ trans('entities.books_sort_name') }}</button>
-        <button type="button" data-sort="created"
+        <button type="button" data-sort="chaptersFirst"
                 class="button outline small">{{ trans('entities.books_sort_created') }}</button>
-        <button type="button" data-sort="updated"
+        <button type="button" data-sort="chaptersFirst"
                 class="button outline small">{{ trans('entities.books_sort_updated') }}</button>
         <button type="button" data-sort="chaptersFirst"
                 class="button outline small">{{ trans('entities.books_sort_chapters_first') }}</button>
-        <button type="button" data-sort="chaptersLast"
+        <button type="button" data-sort="chaptersFirst"
                 class="button outline small">{{ trans('entities.books_sort_chapters_last') }}</button>
     </div>
     <ul class="sortable-page-list sort-list">
