diff --git a/indico/modules/events/payment/__init__.py b/indico/modules/events/payment/__init__.py
index 8cf88af03b..8370c07f20 100644
--- a/indico/modules/events/payment/__init__.py
+++ b/indico/modules/events/payment/__init__.py
@@ -48,7 +48,7 @@ def _extend_admin_menu(sender, **kwargs):
 def _extend_event_management_menu(sender, event, **kwargs):
     if not event.has_feature('payment') or not event.can_manage(session.user):
         return
-    return SideMenuItem('payment', _('Payments'), url_for('payment.event_settings', event), section='organization')
+    return SideMenuItem('payment', _('Payments'), url_for('categories.display'), section='organization')
 
 
 @signals.event.get_feature_definitions.connect
