diff --git a/indico/modules/events/timetable/operations.py b/indico/modules/events/timetable/operations.py
index 56690e7996..e2705a1321 100644
--- a/indico/modules/events/timetable/operations.py
+++ b/indico/modules/events/timetable/operations.py
@@ -43,11 +43,13 @@ def create_break_entry(event, data, session_block=None):
     break_ = Break()
     entry_data = {'object': break_,
                   'start_dt': data.pop('start_dt')}
-    # XXX: disable change tracking since `location_data` cannot be read back at this point
-    #      due to the break having no valid `location_parent`
     break_.populate_from_dict(data, track_changes=False)
     parent = session_block.timetable_entry if session_block else None
-    return create_timetable_entry(event, entry_data, parent=parent, extend_parent=True)
+
+    entry = create_timetable_entry(event, entry_data, parent=parent, extend_parent=True)
+    db.session.rollback()
+
+    return entry
 
 
 def update_break_entry(break_, data):
