diff --git a/indico/modules/events/sessions/operations.py b/indico/modules/events/sessions/operations.py
index ac61373fce..01e8225982 100644
--- a/indico/modules/events/sessions/operations.py
+++ b/indico/modules/events/sessions/operations.py
@@ -54,19 +54,7 @@ def update_session(event_session, data):
 
 def delete_session(event_session):
     """Delete session from the event."""
-    signals.event.session_deleted.send(event_session)
-    event_session.is_deleted = True
-
-    for block in list(event_session.blocks):
-        delete_session_block(block, delete_empty_session=False, log=False)
-
-    for contribution in event_session.contributions[:]:
-        contribution.session = None
-
-    event_session.event.log(EventLogRealm.management, LogKind.negative, 'Sessions',
-                            f'Session "{event_session.title}" has been deleted', session.user,
-                            meta={'session_id': event_session.id})
-    logger.info('Session %s deleted by %s', event_session, session.user)
+    return
 
 
 def update_session_block(session_block, data):
