diff --git a/indico/modules/events/management/controllers/actions.py b/indico/modules/events/management/controllers/actions.py
index 883dc80627..35b2afccf8 100644
--- a/indico/modules/events/management/controllers/actions.py
+++ b/indico/modules/events/management/controllers/actions.py
@@ -28,17 +28,7 @@ class RHDeleteEvent(RHManageEventBase):
         return jsonify_template('events/management/delete_event.html', event=self.event)
 
     def _process_POST(self):
-        self.event.delete('Deleted by user', session.user)
-        flash(_('Event "{}" successfully deleted.').format(self.event.title), 'success')
-        category = self.event.category
-
-        if category and category.can_manage(session.user):
-            redirect_url = url_for('categories.manage_content', category)
-        elif category and category.can_access(session.user):
-            redirect_url = url_for('categories.display', category)
-        else:
-            redirect_url = url_for_index()
-        return jsonify_data(flash=False, redirect=redirect_url)
+        return jsonify_data()
 
 
 class RHChangeEventType(RHManageEventBase):
