diff --git a/indico/modules/events/controllers/creation.py b/indico/modules/events/controllers/creation.py
index 57e5b0ab31..630652e70b 100644
--- a/indico/modules/events/controllers/creation.py
+++ b/indico/modules/events/controllers/creation.py
@@ -153,7 +153,8 @@ class RHCreateEvent(RHProtected):
             else:
                 event = self._create_event(data)
                 notify_event_creation(event)
-            return jsonify_data(flash=False, redirect=url_for('event_management.settings', event))
+            from indico.web.util import url_for_index
+            return jsonify_data(flash=False, redirect=url_for_index())
         check_room_availability = rb_check_user_access(session.user) and config.ENABLE_ROOMBOOKING
         rb_excluded_categories = [c.id for c in rb_settings.get('excluded_categories')]
         category = self._default_category
